x-pack/plugins/monitoring/public/components/apm/apm_metrics.tsx

```diff
@@ -44,6 +44,15 @@ const createCharts = (series: unknown[], props: Partial<Props>) => {
 
 export const ApmMetrics = ({ stats, metrics, seriesToShow, title, ...props }: Props) => {
   const topSeries = [metrics.apm_cpu, metrics.apm_memory, metrics.apm_os_load];
+  const inAgent = stats.config?.container && stats.config?.agentMode;
+
+  const topChartsTitle = inAgent
+    ? i18n.translate('xpack.monitoring.apm.metrics.topCharts.agentTitle', {
+        defaultMessage: 'Elastic Agent Group - Resource Usage',
```

Comment:
The text should probably be changed to `APM & Fleet Server - Resource Usage`, according to the [Messaging for APM on Cloud doc](https://docs.google.com/document/d/1z_AINjDkjHAlq24fqjerKm_njx2ItgfN_jKm1zGTebk/edit#). @nehaduggal is that fine with you?